vm/src/pyobject.rs

```diff
@@ -1308,7 +1339,7 @@ pub trait PyValue: fmt::Debug + PyThreadingConstraint + Sized + 'static {
     fn class(vm: &VirtualMachine) -> PyClassRef;
 
     fn into_simple_object(self, vm: &VirtualMachine) -> PyObjectRef {
```

Comment:
will renaming to `into_exact_object` give more sense to fit to `downcast_exact`?

Reply:
I think this is fine for now; I think if anything `into_object` should be renamed to `into_object_with_class` and `into_simple_object` should be renamed to `into_object`